edx_rbac/utils.py

```diff
@@ -203,8 +207,21 @@ def get_decoded_jwt(request):
     Decodes the request's JWT from either cookies or auth payload and returns it.
     Defaults to an empty dictionary.
     """
-    decoded_jwt = get_decoded_jwt_from_cookie(request) or get_decoded_jwt_from_auth(request)
-    return decoded_jwt or {}
+    if decoded_jwt_from_auth := get_decoded_jwt_from_auth(request):
+        return decoded_jwt_from_auth
+
+    try:
+        if decoded_jwt_from_cookie := get_decoded_jwt_from_cookie(request):
+            return decoded_jwt_from_cookie
+    except InvalidTokenError as exc:
+        logger.info(
+            '[edx_rbac.get_decoded_jwt] Error decoding JWT from cookie and no JWT found in auth: %s',
+            exc,
+        )
+        if not getattr(settings, 'IGNORE_INVALID_JWT_COOKIE', False):
```

Comment:
1. As a top-level setting, should we add an `RBAC_` prefix, or something of that sort?
2. Could you annotate this line following these docs: https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html. You might want to explain that True is the recommended value, and that the default of False is used for backward-compatibility purposes.